website/src/_posts/2020-08-companion-2.0.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Companion 2.0 is here"
+date: 2020-08-20
+author: ife
+published: false
+---
+
+Companion 2.0 is here! 🎉 After maintaining and improving the 1.x series for over a year, we're releasing a Major version bump on the Companion package. So what are the changes you'd expect to see on Companion 2.0?
+
+## Node >= v10
+
+As you must have heard, Node v8.x has reached EOL, and in turn Companion 2.0 drops support for Node v6.x, Node v8.x and now requires that you run at least Node v10.x.
+
+## Renamed Provider Options
+
+Pre 2.0 there were inconsistencies in relation to the Provider names. In some places, the Google Drive provider was referred to as *google* (e.g in `providerOptions`) and in some other places, it was referred to as *drive* (e.g the server endpoints `/drive/list`). In Companion 2.0 it is now made to have the consistent name *drive* everywhere. Similarly the OneDrive provider is made to have the consistent name *onedrive* for the same reason.
+
+## Changed Redirect URIs
+
+On the journey to have more consistent naming with the providers (mentioned above), we have made changes to the redirect URIs that is supplied during the OAuth process. For example, for Google Drive, the old redirect URI was of the form `https://mycompanionwebsite.tld/connect/google/callback`. In Companion 2.0, this is now changed to `https://mycompanionwebsite.tld/drive/redirect`. This is a Breaking Change and it means you would need to make these corresponding changes to your redirect URIs on your Providers' API Dashboards.
```

Comment:
I think this first sentence is also a bit lengthy but I don't have a good suggestion to improve it, so maybe we should ask AJ for some expert editing later :)

Reply:
I rephrased it a bit. Please let me know if you think it still needs adjustment